scripts/api_dump_generator.py

```diff
@@ -1500,6 +1500,9 @@
 {{
     char *tm = (char*)t;
     size_t tmlen;
+#ifdef __ANDROID__
+    (void)tmlen; // Appears to be unused on Android
```

Comment:
@charles-lunarg I'm not familiar with code generation here, but--on Android at least--the `@foreach union` block below appears to not produce anything, causing an "unused variable warning" here.

Reply:
oops!
I haven't touched the android stuff in api_dump in forever. Feel free to make it work as you see fit.

Reply:
After further review & discussion, this is a bug. The unions used should be from the Vulkan API, not the Video API. Thus, I will make a PR to fix it shortly.

Reply:
This has been fixed, please rebase on main to see the changes if you haven't already.